lib/main.mlx

```diff
@@ -205,6 +205,11 @@ let q_and_a = [
 [JSX.string "$ opam pin add dune --dev-repo"]
 </Code>
     </div>
+  ; "How does the nightly differ from the preview?",
+    <div class_="flex flex-col gap-2">
+      <p>"Like the nightly, the preview was a binary build created every day from the most current Dune sources. However, it did enable certain optional features by default. These features can still be used in the nightly, but they have to be enabled at runtime."</p>
```

Comment:
I think it's unhelpful to be generic about this. We can be more specific here. There were 3 features enabled before, let's talk a bit about how they got absorbed. The pkg progress is enabled from `--display short`. Automatic dev tool installation is hidden behind `DUNE_CONFIG__dev...` (I don't remember). And portable lock directories are now the default behaviour.

Reply:
That happened during the course of the preview so probably doesn't concern us here.